src/aggregated_bls/party_i.rs

```diff
@@ -1,17 +1,22 @@
-use crate::aggregated_bls::h1;
-use crate::basic_bls::BLSSignature;
+use std::collections::HashSet;
+
 use curv::arithmetic::traits::Modulo;
 use curv::elliptic::curves::bls12_381::g1::FE as FE1;
 use curv::elliptic::curves::bls12_381::g1::GE as GE1;
 use curv::elliptic::curves::bls12_381::g2::FE as FE2;
 use curv::elliptic::curves::bls12_381::g2::GE as GE2;
+use curv::elliptic::curves::bls12_381::Pair;
 use curv::elliptic::curves::traits::ECPoint;
 use curv::elliptic::curves::traits::ECScalar;
 use curv::BigInt;
+use pairing_plus::bls12_381::Bls12;
```

Comment:
we should be able to use only functions from Curv library (see comment below to how to use Curv API for pairings)